packages/types/Calendar.d.ts

```diff
@@ -215,7 +215,7 @@ export interface IntegrationCalendar extends Ensure<Partial<SelectedCalendar>, "
 }
 
 export interface Calendar {
-  createEvent(event: CalendarEvent): Promise<NewCalendarEventType>;
+  createEvent(event: CalendarEvent): Promise<NewCalendarEventType | undefined>;
```

Comment:
This was already a valid value but it was not here so it couldn't be defined as `undefined` for non-traditional calendar apps such as this.